Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -334,7 +334,30 @@ PTF_TEST_CASE(TestPcapLiveDeviceNoNetworking)
 
 } // TestPcapLiveDeviceNoNetworking
 
+PTF_TEST_CASE(TestPcapLiveDeviceNoNetworkingIPv6)
```

Comment:
This test assumes the machine running it has a NIC with IPv6 address. This is not always the case, for example in the VMs running the CI: https://github.com/seladb/PcapPlusPlus/actions/runs/3068629818

There is no easy way around it because we need a live device to test this functionality.

One option I can think of is [creating a virtual interface](https://linuxconfig.org/configuring-virtual-network-interfaces-in-linux) but it's only available on Linux and won't work on other platforms.

Another option that requires much more work is creating a mock library for libpcap/WinPcap/Npcap where we can mock everything these libraries do and then have full control, but it's probably out-of-scope for this PR.

Unfortunately, as I see no other option, I suggest we omit the test for this functionality

Reply:
@seladb at this stage, I omitted the test for the feature. maybe later we can develop about mock library or any other solution. thank you